pages/docs/concepts/message.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Message
+weight: 2
```

Comment:
```suggestion
weight: 5
```

We can't use `2` as your **weight** for this NEW Doc page because your currently merged `server` page already has that weight. Def a good idea to check the latest weight used in the previous pages in each section so that you always know what number to start ordering with. 😄✌🏽